apps/server/lib/lexical/server/code_intelligence/completion/translations/macro.ex

```diff
@@ -438,6 +438,58 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Macro do
     |> builder.boost()
   end
 
+  def translate(%Candidate.Macro{name: "test", arity: 1} = macro, builder, env) do
```

Comment:
Overall here, I think the code is mostly fine, but the UX needs help. Some comments

For whatever reason, I don't think about test with regard to arity, so the `/arity` label isn't helping.  Similarly, the argument names are just `message`, `var` and `contents` for the `/3` function, so that's not helpful. 

What I found worked best was giving them labels like:
```elixir
  stub_label = ~S(test "message"           )
  plain_label = ~S(test "message" do...     )
  context_label = ~S(test "message", %{} do...)
```
One thing to note is that the labels are the same length, so they stack nicely. I also gave them details like "A stub test", "A test" and "A test with a context variable". Which helped to disambiguate.

Reply:
Updated, I thought `A test that receives context` sounds a bit more specific